README.md

```diff
@@ -48,7 +48,15 @@ When building on Linux it is sometimes necessary to run:
 
     $ gradle wrapper
     
-Finally, turn on the "Use the new formplayer frontend" feature flag on your CommCareHQ domain
+### Running in IntelliJ
+
+In order to set breakpoints, step through code, and link the runtime with source you'll need to run the code in an IDE. We use IntelliJ. To setup
+
+1. Download [IntelliJ IDE](https://www.jetbrains.com/idea/download/#section=mac)
+2. Open IntelliJ and select "Import Project"
+3. Navigate to the cloned `formplayer` repository and select `build.gradle` at the root
+4. De-select "Use auto-import" and "Create directories for empty content roots automatically" and *select* "Use gradle wrapper"
+5. Click "OK"
 
```

Comment:
@wpride I understand that it's not possible to enumerate all of the finicky things you had to do in IntelliJ after this point to get it to work, but for the sake of transparency and not leaving people thinking they did something wrong, it might make sense to add a line here that says something like "You may need to do some further configuring of your own IDE settings to get everything building properly from here." Thoughts?